.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,28 @@
+name: Update dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * 1'  # "At 06:00 on Monday."
+
+jobs:
+  update-dependencies:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: excitedleigh/setup-nox@1.0.0
+    - name: "Run update: dependencies"
+      run: pipx run nox
+    - name: Create Pull Request
+      if: github.ref == 'refs/heads/master' && github.repository == 'pypa/manylinux'
+      uses: peter-evans/create-pull-request@v3
```

Comment:
FYI, `gh` could probably do this as well, see https://github.blog/2021-03-11-scripting-with-github-cli/#using-gh-in-github-actions

Reply:
Good to know.
I think I'll get a look at `gh` when updating the workflow to check more dependencies.
For now the simple `peter-evans/create-pull-request@v3` is enough I guess.